hack/generator/pkg/astmodel/resource.go

```diff
@@ -218,7 +222,6 @@ func (definition *ResourceType) RequiredImports() []PackageReference {
 
 	typeImports = append(typeImports, MetaV1PackageReference)
 	typeImports = append(typeImports, MakeGenRuntimePackageReference())
-	typeImports = append(typeImports, MakePackageReference("fmt"))
```

Comment:
Removing this hard coded reference needed so the new storage types compile; the Arm functions generate the reference they need anyway.